lib/api.js

```diff
@@ -116,5 +116,39 @@ module.exports = {
       body: tx.toJSON()
     })
     .then(r => r.json())
-  }
+  },
+  
+  getTransactionOwnerships: function (address, endpoint) {
```

Comment:
This API may be added in index.js to be accessible from library

Reply:
👍